extensions/wikia/LookupUser/LookupUser.body.php

```diff
@@ -165,26 +165,27 @@ function showInfo( $target, $emailUser = "" ) {
 			}
 
 			// Check for disabled accounts where we kept the email
-			$dRows = $dbr->select(
-				[ '`user`', 'user_properties' ],
-				[ 'user_name' ],
+			$userIds = $dbr->selectFieldValues(
+				'user_properties',
+				'up_user',
 				[
-					'user_id = up_user',
 					'up_property' => 'disabled-user-email',
 					'up_value' => $target,
 				],
 				__METHOD__
 			);
 
-			/** @var stdClass $row */
-			foreach ( $dRows as $row ) {
+			foreach ( User::whoAre( $userIds ) as $user_id => $user_name ) {
+				// User::whoAre return an entry for anons (under '0' key), skip it here
+				if ( $user_id === 0 ) continue;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)

[project-id]: wikia:app
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') 2 statements were found on this line. Reformat the code to have only one statement per line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)

[project-id]: wikia:app